src/api/java/com/minecolonies/api/colony/IColonyTagCapability.java

```diff
@@ -144,7 +147,7 @@ public NBTBase writeNBT(@NotNull final Capability<IColonyTagCapability> capabili
         {
             final NBTTagCompound compound = new NBTTagCompound();
             compound.setInteger(TAG_ID, instance.getOwningColony());
-            compound.setTag(TAG_COLONIES, instance.getAllCloseColonies().stream().map(id -> Storage.write(id)).collect(NBTUtils.toNBTTagList()));
+            compound.setTag(TAG_COLONIES, instance.getAllCloseColonies().stream().map(Storage::write).collect(NBTUtils.toNBTTagList()));
```

Comment:
I do not think that Storage::write makes sense here. Either rename the class or reference to what it is a Storage for, or make a comment what Storage this is. Cause if this is a Cap storage it has no reason to be here.

Reply:
Storage is the sub class it is in, and write is a static method.